js/Controller.js

```diff
@@ -192,6 +188,9 @@ class Controller{
             // Finding the royal diamonds is the win condition, so it does these special things
             this.charmodel.whiffles = 999999999;	// "one zillion zillion whiffles"
             var winMessage = "You found the royal diamonds! Victory Royale!";
+            this.mapmodel.godMode = true;
```

Comment:
This worked fine on Chrome, which is the only browser we'll demo on, so everything is fine.
On Firefox, this created problems, FYI, because godMode takes effect and turns the whole map Meadow green during initialization... but then the alert(winMessage) prevents godMode from redrawing properly later, and the map appears Meadow green until you dismiss the dialog. 
Anyway, it doesn't matter because we're Chrome-only for the demo.

Reply:
Just updated this with an immediate redraw after calling createMap so it fixes it in Firefox. Thanks for pointing that out.